Movies_app/Movies_app/Coordinator/MoviesListCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+// MoviesListCoordinator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
+
+final class MoviesListCoordinator: BaseCoordinator {
+    // MARK: - Public properties
+
+    var rootController: UINavigationController?
```

Comment:
```suggestion
   private var rootController: UINavigationController?
```
и это Visual Components